lib/Auth/Process/PerunEntitlement.php

```diff
@@ -154,6 +165,17 @@ private function getForwardedEduPersonEntitlement(&$request)
 
     private function getResourceCapabilities(&$request)
     {
+        $capabilitiesResult = [];
+
+        if (!isset($request['perun']['groups'])) {
+            Logger::error(
+                'perun:PerunEntitlement: ' .
+                'missing mandatory field \'perun.groups\' in request.' .
+                'Hint: Did you configure PerunIdentity filter before this filter?'
```

Comment:
I would change the hint to: _"Did you configure PerunIdentity filter to run before this filer?_